test/models/podcast_planner_test.rb

```diff
@@ -0,0 +1,423 @@
+require "test_helper"
+
```

Comment:
new tests for the `PodcastPlanner` model. did not test every single small method, but focused on the larger more complex ones that utilize the smaller ones.